### PR TITLE
Update production node

### PR DESCRIPTION
**Pull request by jessie-ross**
Likely fix for #889

## How has this been tested?

- Automated tests that cover added/changed functionality

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # Use long term support version of Node.js
-FROM node:14.16.1
+FROM node:16.18.1
```
Comment (withinboredom) [inline Dockerfile:2]: Might as well go for LTS...

Comment (aaronyan) [reply Dockerfile:2]: Woah, hi there @withinboredom! @jessie-ross, what would be required to move to [v18.12.0 (LTS)](https://nodejs.org/en/blog/release/v18.12.0)?

Comment (jessie-ross) [reply Dockerfile:2]: Haha hi!!!

If we move to LTS we'll run into this again ;)

Screenshot 2023-07-19 at 1 50 15 pm

Comment (jessie-ross) [reply Dockerfile:2]: We could move to 18, maybe as a separate PR though.

Comment (jessie-ross) [reply Dockerfile:2]: We should move to 18, the maintenance window is closing on 16 soonish

Comment (jessie-ross) [reply Dockerfile]: I'm trying it in this PR

**Review by aaronyan — APPROVED** [review]
LGTM thank you!

**Pull request merged**